skipper.go

```diff
@@ -1211,6 +1211,7 @@ func (o *Options) tlsConfig(cr *certregistry.CertRegistry) (*tls.Config, error)
 
 	if cr != nil {
 		config.GetCertificate = cr.GetCertFromHello
+		config.GetConfigForClient = cr.GetConfigFromHello
```

Comment:
problem: the `tls.Config` from `GetConfigForClient` will override the previously used `GetCertificate`.  So the two are mutually exclusive, from the docs:

> Server configurations must set one of Certificates, GetCertificate or GetConfigForClient.

Reply:
Check if the new one is non nil and if so only use that one, else fallback to current behavior.
Does it make sense?

Reply:
Makes sense.

I was also thinking we could also use `GetConfigForClient` only. This would allow us to use custom certs and custom TLS config. Config could be set to a "default" based on global skipper flags allowing it to be overridden. Wdyt?